nativelib/src/main/scala/scala/scalanative/unsafe/package.scala

```diff
@@ -196,30 +196,54 @@ package object unsafe extends unsafe.UnsafePackageCompat {
     }
   }
 
+  private def toCStringNonNull(cb: CharBuffer, cs: Charset, charSize: CInt)(
+      implicit z: Zone
+  ): Ptr[Byte] = {
+    val buf = cs.encode(cb)
+    val off = buf.position()
+
+    val size = (buf.limit() - off).toUSize
+    val cstr = z.alloc(size + charSize.toUSize)
+    ffi.memcpy(cstr, buf.array().at(off), size)
+
+    // Set null termination bytes
```

Comment:
This can become something like:
// null termination byte/bytes already set by z.alloc

Just in case a reader gets there and wonders because
they skimmed past the comment at z.alloc.

Reply:
Comment amended.